### DISCUSSION TITLE
Could you all add me as a collaborator here?

### DISCUSSION THREAD

**Discussion by ghost** [jnode-revisited/jnode#18] [Q&A] [answered]
Hi @tripleo1, how are you?

I would like to help a lot here, a simple idea I had would be to add this git implementation in java to jnode: [Curiouskid0423/Git](https://github.com/Curiouskid0423/Git), [gitlet](https://github.com/dylan-goetting/gitlet)

could you all add me as a collaborator here?

**Comment by tripleo1** [comment, marked as answer]
Hi, I am well.

If you can figure out where to put it, send a PR, and I'll review it.

thx
